### PR TITLE
Python

### PR DESCRIPTION
**Pull request by travis032654**

### CODE REVIEW

File: Templates/Python/Model/ComplexType.py.tt [hunk 1]
```diff
@@ -86,11 +86,11 @@ class <#=complex.Name.ToUpperFirstChar()#>(OneDriveObjectBase):
         Gets and sets the <#=property.Name#>
         
         Returns: 
-            :class:`<#=property.Type.GetTypeString()#><onedrivesdk.model.<#=property.Type.GetTypeString().ToUnderscore()#>.<#=property.Type.GetTypeString()#>>`:
+            :class:`<#=property.Type.GetTypeString()#><microsoft.graph.model.<#=property.Type.GetTypeString().ToUnderscore()#>.<#=property.Type.GetTypeString()#>>`:
```
Comment (shiftylogic) [inline Templates/Python/Model/ComplexType.py.tt]: There shouldn't be any hard coded namespaces. This should have come out of the Vipr object models.

File: Templates/Python/Model/EntityType.py.tt [hunk 2]
```diff
@@ -50,7 +57,7 @@ class <#=entity.Name.ToUpperFirstChar()#>(OneDriveObjectBase):
         """Gets and sets the <#=property.Name#>
         
         Returns: 
-            :class:`<#=propertyNameCap#>CollectionPage<onedrivesdk.request.<#=(propertyNameCap+"Collection").ToUnderscore()#>.<#=propertyNameCap#>CollectionPage>`:
+            :class:`<#=propertyNameCap#>CollectionPage<microsoft.graph.request.<#=(propertyNameCap+"Collection").ToUnderscore()#>.<#=propertyNameCap#>CollectionPage>`:
```
Comment (shiftylogic) [inline Templates/Python/Model/EntityType.py.tt]: Another hard coded namespace.

Comment (shiftylogic) [inline Templates/Python/Model/EntityType.py.tt]: See other places as well.

**Comment by shiftylogic** [comment]
I created a new python branch on this repo. Can you please move this entire pull request over to the python branch?

**Comment by iambmelt** [comment]
I'm unclear as to why this PR would contain any changes to `Templates/Android` - I suspect these may be artifacts from a bad merge. Recommend we not merge these changes at this time.

**Pull request closed without merging**